game.py

```diff
@@ -118,10 +118,10 @@ def __init__(self, canvas, parent=None, path=None):
                     'background2.jpg']
         self._backgrounds = []
         for bg in self._BG:
+            pixbuf = GdkPixbuf.Pixbuf.new_from_file(os.path.join(self._path, 'images', bg))
+            pixbuf = pixbuf.scale_simple(self._width, self._height, GdkPixbuf.InterpType.BILINEAR)
             self._backgrounds.append(Sprite(
-                    self._sprites, 0, 0, GdkPixbuf.Pixbuf.new_from_file_at_size(
-                        os.path.join(self._path, 'images', bg),
-                        self._width, self._height)))
+                self._sprites, 0, 0, pixbuf))
```

Comment:
Can you explain why this change?

Reply:
The changes is to fix activity resolution, and make it compatible with all screens as it enlarges image according to screen height and width and set it.  GdkPixbuf.InterpType.BILINEAR is  interpolation mode, it has a good speed/quality balance.

Reply:
I don't think this changes anything functionally. `GdkPixbuf.Pixbuf.new_from_file_at_size()` function call immediately resizes the image after it loads from file. While `GdkPixbuf.InterpType.BILINEAR` is also used to resize image. 
Also, please tell me what is your test resolution? Did you face an issue before that making this change fixed?

Reply:
Thanks for looking. I don't understand completely btw why this change fixed the problem but as earlier It was n't working properly for large display sizes as look [here](https://github.com/sugarlabs/locosugar/pull/14#issuecomment-614414061). The problem can be according to me is that, the image we are using here is of 1200x900 pixel size, so It won't work on high resolution devices as it will automatically load 1200x900 image size, it won't scale the image. We here need to scale the image to screen size . I don't know I could be wrong. But If you can check on a device of high resolution please test there. Because I got the problem fixed there now after these changes. Inform me if I am wrong.

Reply:
line 124, mentions the size we want the image to load to. From what I understand, `GdkPixbuf.Pixbuf.new_from_file_at_size` is supposed to load the image and rescale it to the given dimensions.